specification/compute/resource-manager/readme.md

```diff
@@ -184,6 +184,7 @@ These settings apply only when `--tag=package-compute-only-2018-06` is specified
 ``` yaml $(tag) == 'package-compute-only-2018-06'
 input-file:
 - Microsoft.Compute/stable/2018-06-01/compute.json
+- Microsoft.Compute/stable/2018-06-01/runCommands.json

```

Comment:
Please attach C# SDK test validation proof, because this is what we all do.

Reply:
I don't understand? This is a Swagger question, I don't do C# and I'm not in the C# team, I'm in the Swagger team. This PR is just a friendly reminder that this is needed in order to ship CLI with 2018-06-01 support.

Reply:
I can understand your workflow is to patch the specification, and SDK release is not your concern. But if you can waive the SDK proof part, can we also get a waiver onwards? @hyonholee

Reply:
Let's continue this by email, thank you